src/woo_publications/publications/api/serializers.py

```diff
@@ -641,6 +658,8 @@ def create(self, validated_data):
                     f"Unexpected creation publicatiestatus: {publicatiestatus}"
                 )
 
+        publication.save()
+
         publication.apply_retention_policy()
```

Comment:
Extra `save()`? Maybe remove the save from the apply and save afterwards? Or make the apply function accept a `save: bool=True`. If that makes more sense from the other call sites.